cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+
+/**
+ * Match values at a location
+ * @param {string} location target where to search for string.
+ * @param {string} matchValue target area string to compare with.
+ */
+Cypress.Commands.add( 'MatchValue', ( location, matchValue ) => {
```

Comment:
Why are the functions `Capitalised` vs `camelCased`?

Reply:
All the functions were this way. Since there are commands and to keep them consisted I `Capitalized` them

Reply:
How do you mean? I saw ` cy.visit` and other commands in `camelCase`